lib/recording/RecordingEntrySession.js

```diff
@@ -132,20 +189,21 @@ class RecordingEntrySession {
         return Q.resolve();
     }
 
-    handleReachedMaxRecordingDuration() {
-        if (this.entryObject.serverType == kalturaTypes.KalturaEntryServerNodeType.LIVE_PRIMARY) {
+    splitRecordedEntry(entryObject = this.entryObject) {
```

Comment:
this is not good, please don't give such default properties, it's bit unpredictable what is the value